reg_tests/CMakeLists.txt

```diff
@@ -181,7 +181,7 @@ add_custom_target(
 )
 
 add_custom_target(
-  regression_tests
+  regression_tests ALL
```

Comment:
It might be good to also add `ALL` to the `regression_test_module_drivers` target below to be consistent.

Reply:
done!